Documentation/docs/source/installation.rst

```diff
@@ -120,7 +120,7 @@ If the latest SimpleITK bug fixes or features are required then
 pre-release binaries are needed. The pre-release binaries are built
 from the actively developed master branch. The SimpleITK packages are
 available on `Github releases
-<https://github.com/SimpleITK/SimpleITK/releases/tag/latest>`__ under
```

Comment:
This is the one link that should remain to the latest.

Reply:
Reverted this and force pushed, so resolved.